ads/sovrn.js

```diff
@@ -16,19 +16,21 @@
  * Existing sovrn customers feel free to contact amp-implementations@sovrn.com for assistance with setting up your amp-ad tagid
  * New customers please see www.sovrn.com to sign up and get started!
  *********
+ **/
 /**
  * @param {!Window} global
  * @param {!Object} data
  */
-export function sovrn(global, data) {
+ import {writeScript} from '../src/3p';
```

Comment:
`import` should go above the comment with `@param` since that jsdoc documents the function itself.